includes/admin/emails/class-email-notification-util.php

```diff
@@ -285,6 +285,12 @@ public static function get_value( Give_Email_Notification $email, $option_name,
 			? str_replace( '_give_', '', $option_name )
 			: $option_name );
 		$option_value       = give_get_option( $global_option_name, $default );
+		
+		//Emails sent to donors cannot be translated using WPML.
```

Comment:
Please update comment to:

`// Ensure that emails sent to donors can be translated using WPML`

Reply:
Done